src/components/visual-tooltip/VisualTooltip.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+
+import Tooltip, { TooltipProps, TooltipTheme } from '../tooltip';
+
+import './VisualTooltip.scss';
+
+// We manually set "text" with our specific visual tooltip content.
+type OtherTooltipProps = Omit<TooltipProps, 'text'>;
```

Comment:
seeing some issues importing `TooltipProps`:

```
WARNING in ./src/components/tooltip/index.ts 1:0-81
"export 'TooltipProps' was not found in './Tooltip'
 @ ./src/icons/icons.stories.tsx
 @ ./src sync \.stories\.(js|tsx)$
 @ ./.storybook/config.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./node_modules/@storybook/addon-docs/dist/frameworks/react/config.js ./.storybook/config.js ./node_modules/webpack-hot-middleware/client.js?reload=true&quiet=true
```

I think it's an issue with webpack and the TS interfaces. Need to explore this more.